packages/backend/models/unified/chatUsers.ts

```diff
@@ -7,9 +7,9 @@ export interface UnifiedChatUser {
 
 export function unifyChatUser(user: any): UnifiedChatUser {
     const unifiedUser: UnifiedChatUser = {
-        id: user.id,
-        name: user.real_name,
-        createdTimeStamp: new Date(user.updated * 1000).toISOString(),
+        id: user.id || user.user.id,
+        name: user.profile.real_name || user.user.global_name,
```

Comment:
This will throw if `user.profile`or `user.user`is `undefined`. Please handle those cases.